test/models/work_test.rb

```diff
@@ -0,0 +1,63 @@
+require "test_helper"
+
+describe Work do
+  let(:work) { Work.new }
+  let(:book) { works(:book1)}
+  let(:album1) {works(:album1)}
+
+  describe "relationships" do
+    it 'can have votes' do
+      album1.votes.count.must_equal 2
+    end
+  end # Relationships
+
+  describe "validations" do
+    it "must have a title to be valid" do
+      #false
+      work.category = "album"
+      work.valid?.must_equal false
+      work.errors[:title].must_include "can't be blank"
+
+      #true
+      work.title = "A Title"
+      work.valid?.must_equal true
+    end
+
+    it "must have a category to be valid" do
+      #false
+      work.title = "A Title"
+      work.valid?.must_equal false
+      work.errors[:category].must_include "can't be blank"
+
+      #true
+      work.category = "album"
+      work.valid?.must_equal true
+    end
+
+    it "must have a unique title in relation to its category to be valid" do
+      # false
+      new_book = Work.new(title: book.title, category: book.category)
+      another_book = Work.new(title: book.title, category: book.category)
+
+      (new_book && another_book).valid?.must_equal false
```

Comment:
I don't think this does what you think it does.  

Instead `(new_book && another_book)` returns `another_book` which has `valid?` called on it.  It doesn't use `new_book` at all.  

Instead you should have 
```
another_book.valid?.must_equal false
new_book.valid?.must_equal false
```